docs/src/gcode/overview.adoc

```diff
@@ -968,8 +968,7 @@ control.
 [[gcode:comments]]
 == Comments(((Comments)))
 
-Comments are purely informative and have no
-influence on machine behaviour
+Comments are purely informative and have no influence on machine behaviour.
```

Comment:
What is the purpose of this change?  It look like it should have no effect on asciidoc nor po4a.

Reply:
Asciidoc eye candy. Important to me when I see it.

Reply:
Will that be a blocker for this PR? :smile:  :innocent: